README.md

```diff
@@ -1,20 +1,453 @@
+
+```js
+pikachu["name"] // "Pikachu"
+pikachu["type"] // "Electric"
+pikachu["species"] // "Mouse"
+pikachu["level"] // 100
+```
+
+`pikachu["level"]` is much easier to understand than `pikachu[3]` because the key gives us some context.
+
+Each key on the Object is also called a `property` of the Object. An Object's properties can also be accessed using dot notation:
+
+```js
+pikachu.speak(); // "pika pika"
+```
+
+This is equivalent to using the key string:
+```js
+pikachu["speak"](); // "pika pika"
+```
+
+Notice how we stored many properties of different types on the pikachu Object. Objects are like containers for storing variables. An Object can store any type of variable such as a primitive, functions or even other Objects. Objects have many uses besides representing a single entity such as a Pokemon. The main idea behind Objects is that they provide a way to group variables that are easily accessible via descripitve keys.
+
+
+### Primer on Prototypes
+Throughout this lesson you'll see the terms `prototype` and `prototype chain` a few times. The prototype is like DNA. Its passed down from parent to child, to grandchild...etc. The prototype chain refers to the list of all ancestors (parents, grandparents, great grandparents...etc). Just as a child would inherit properties from their parents, Objects and other JavaScript types inherit properties from Object. Think of Object as the oldest living relative from which all other types inherit their DNA from. Unlike human properties such as eye color, hair color..etc, these properties are functions, strings, numbers... any type you can think of. Properties that exist on the child but not on the parent are called `direct properties`.
+
+## Creating Objects
+
+So far we've seen one way to create an Object but there are a few more.
+
+
+#### Using Object literals
+In our Pokemon example, we created an Object using curly braces. This is called the Object literal syntax:
+
+```js
+var obj = {};
+```
+
+Objects created using the Object literal syntax can have key-value pairs passed in between the braces as seen in the pikachu example above or have values added to them after creation as we'll see later on in the [Adding Properties To Objects](#adding-properties-to-objects) section.
+
+#### Using the Object constructor
+
+The keyword `Object` is a function that when invoked, returns an instance to a new Object.
+
+```js
+var obj = new Object();
+```
+
+#### Using Object.create method
+
+`Object.create` is a method for creating Objects with a specific prototype. Passing `null` will create an Object that has no prototype. This can be useful when we don't care about inherting properties from the prototype chain.
```

Comment:
This might be a little advanced for students at this stage, simply because they haven't practiced working with prototypes much yet. Is there a gentler way to ease them into `Object.create()`?